system/gatekeeper/templates/constrainttemplate-restrict-monsoon3-deployments.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: templates.gatekeeper.sh/v1
+kind: ConstraintTemplate
+metadata:
+  name: gkrestrictmonsoon3deployments
+spec:
+  crd:
+    spec:
+      names:
+        kind: GkRestrictMonsoon3Deployments
+      validation:
+        openAPIV3Schema:
+          type: object
+          properties:
+            patterns:
+              type: array
+              items:
+                description: regex pattern that release names must match
+                type: string
+              minItems: 0
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
+        package restrictnewdeployments
+
+        import future.keywords.in
+
+        iro = input.review.object
+        n := iro.metadata.namespace
+        releaseName := iro.metadata.name
+
+        violation[{"msg": msg}] {
+          n == "monsoon3"
```

Comment:
I just realized that this needs to apply to the namespace `monsoon3global`. (That's where monsoon3 components for the "global" region are deployed to.)